addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -529,8 +529,8 @@ export class HtmlField extends Component {
             ids: attachment
         }]));
     }
-    _onWysiwygBlur() {
-        this.commitChanges({ urgent: true });
+    async _onWysiwygBlur() {
+        await this.commitChanges({ urgent: true });
```

Comment:
I'm confused. This callback doesn't look to be awaited anyway so this should not have any impact. Did I miss something ? cc @Zinston Because it looks related to the bug we discussed the other day with the convert inline at blur.

Reply:
it was just a test because there's a bug in the html field, but fge is working on it I think (so this commit will be dropped)

Reply:
@std-odoo I wanted to ping fge but it looks like his github username is not fge-odoo.

Reply:
Yes I guess we're talking about the same bug. I'd be interested to know more about fge's work on it (which aspects he tackles and how). I can share my notes on the subject too :) Glad to know that someone at JS team is already working on it, we were going to come by today to talk about it!

Reply:
@FrancoisGe found you :-D

Reply:
@Zinston  How can I help you?

Reply:
@FrancoisGe just if you have a task/pr I'd be interested in taking a look.

Reply:
> @FrancoisGe just if you have a task/pr I'd be interested in taking a look.

Are you talking about the bug that we have spot in knowlegde ? An udpate that is made after the destruction of the view ?

Reply:
Yes that's excatly this bug :) (but it come from the html field itself)

Reply:
So I fixed the crash. But we find it very strange that an onchange is triggered after the destruction of a view in a normal flow (not the closing of the tab). We don't think it should happen.

Reply:
Is it the task linked to this message ? https://discord.com/channels/678381219515465750/687338152372994236/1044213239862538310
Do you have a PR ? (there's no crash anymore, but still a warning because of issue in the html field of some async methods not awaited)
@abd-msyukyu-odoo  well described the issue

Reply:
This is my fix https://github.com/odoo/odoo/pull/106135.
Before this one, we had a crash when we created a new Knowledge page with only 1 space and when we changed page. The problem is that you should never do an onchange in this kind of use case after the destruction of the view .

Reply:
Ah ok, but it doesn't fix the issue in the html field itself...
I'll just cherry-pick the original abd fix: https://github.com/odoo/odoo/pull/105762/files